sdk/cosmos/azure-cosmos-spark_3-2_2-12/CHANGELOG.md

```diff
@@ -3,12 +3,14 @@
 ### 4.26.0-beta.1 (Unreleased)
 
 #### Features Added
+* Added config option `spark.cosmos.http.connectionPoolSize` to override the Http Connection Pool size in Gateway mode. Increasing the connection pool size beyond 1000 can be useful when the number of concurrent requests in Gateway mode is very high and you see a `reactor.netty.internal.shaded.reactor.pool.PoolAcquirePendingLimitException: Pending acquire queue has reached its maximum size of 2000` error. - See [PR 38305](https://github.com/Azure/azure-sdk-for-java/pull/38305)
 
 #### Breaking Changes
 
 #### Bugs Fixed
 
 #### Other Changes
+* Reduced CPU overhead slightly for workloads with high throughput of point operations - especially when diagnostics like traces or metrics are enabled. - See [PR 38232](https://github.com/Azure/azure-sdk-for-java/pull/38232) 
```

Comment:
We could specify this in the changelog for `azure-cosmos` and the hotfixed `azure-cosmos-spark` can take a dependency on hotfixed `azure-cosmos`?